ddtrace/vendor/lightstep/constants.py

```diff
@@ -0,0 +1,8 @@
+ACCESS_TOKEN_ENV_VAR = "LIGHTSTEP_ACCESS_TOKEN"
+ACCESS_TOKEN = "lightstep.access_token"
+
+COMPONENT_NAME = "lightstep.component_name"
+COMPONENT_NAME_ENV_VAR = "LIGHTSTEP_COMPONENT_NAME"
+
+SERVICE_VERSION = "service.version"
```

Comment:
should this be named `lightstep.service_version`?

Reply:
`service.version` is correct here